### PR TITLE
Viktig tillegg

### PR DESCRIPTION
**Pull request by sondrewb**
La til viktige ting

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -9,3 +9,9 @@ Helloyisen!
 
 
 It is very fun to use git.
+
+(∩ ͡° ͜ʖ ͡°)⊃━☆ﾟ. * ･ ｡ﾟYou've been touched by the donger fairy
```
Comment (stianjensen) [inline README.md]: U wot m8 ill rek u 1v1 me scrub

Comment (sondrewbaugsto) [inline README.md]: ┌༼ຈل͜ຈ༽┘test donger, please ignore ┌༼ຈل͜ຈ༽┘

**Comment by aleksanb** [comment]
Edit: Closed, really?

**Comment by aleksanb** [comment]
I was depending on this feature in my private fork of this repo.
SAD to not see it upstreamed.

**Pull request closed without merging**